src/hooks/useLayoutEffect.ts

```diff
@@ -0,0 +1,5 @@
+import * as React from 'react';
+
+const isBrowser = typeof window !== 'undefined' && window.document && window.document.createElement;
```

Comment:
Use this: https://github.com/react-component/util/blob/master/src/Dom/canUseDom.ts

Reply:
@zombieJ ok